clients/include/blas1/testing_iamax_iamin_strided_batched.hpp

```diff
@@ -245,9 +257,16 @@ inline void testname_iamin_strided_batched(const Arguments& arg, std::string& na
 template <typename T>
 void testing_iamin_strided_batched(const Arguments& arg)
 {
-    bool FORTRAN = arg.api == hipblas_client_api::FORTRAN;
-    auto hipblasIaminStridedBatchedFn
-        = FORTRAN ? hipblasIaminStridedBatched<T, true> : hipblasIaminStridedBatched<T, false>;
-
-    testing_iamax_iamin_strided_batched<T, ref_iamin<T>>(arg, hipblasIaminStridedBatchedFn);
+    auto hipblasIaminStridedBatchedFn    = arg.api == FORTRAN ? hipblasIaminStridedBatched<T, true>
+                                                              : hipblasIaminStridedBatched<T, false>;
+    auto hipblasIaminStridedBatched_64Fn = arg.api == FORTRAN_64
+                                               ? hipblasIaminStridedBatched_64<T, true>
+                                               : hipblasIaminStridedBatched_64<T, false>;
+
+    if(arg.api & c_API_64)
+        testing_iamax_iamin_strided_batched<T, ref_iamin<T>, int64_t>(
+            arg, hipblasIaminStridedBatched_64Fn);
+    else
+        testing_iamax_iamin_strided_batched<T, ref_iamin<T>, int>(arg,
+                                                                  hipblasIaminStridedBatchedFn);
```

Comment:
This collides a bit with what I did for the reference but should should be easy to resolve.